packages/app/src/app/pages/Sandbox/Editor/Workspace/Chat/index.tsx

```diff
@@ -54,76 +64,84 @@ export const Chat: React.FC = () => {
   };
 
   const { messages, users } = state.live.roomInfo.chat;
-  const currentUserId = state.live.liveUserId;
   const roomInfoUsers = state.live.roomInfo.users;
 
+  const orderedMessages: (m: any) => any[] = m =>
+    sortBy(takeRight(m, 100), 'date');
+
+  const messageData = message => {
+    const metadata = roomInfoUsers.find(u => u.id === message.userId);
+    return {
+      metadata,
+      color: metadata
+        ? `rgb(${metadata.color[0]}, ${metadata.color[1]}, ${metadata.color[2]})`
+        : '#636363',
+      name: users[message.userId],
+    };
+  };
+
+  const isNotSameUser = (message, i) =>
+    i === 0 || messages[i - 1].userId !== message.userId;
+
   return (
-    <Container ref={messagesRef}>
-      <Messages>
-        {messages.length > 0 ? (
-          sortBy(takeRight(messages, 100), 'date').map((message, i) => {
-            const metadata = roomInfoUsers.find(u => u.id === message.userId);
-            const color = metadata
-              ? `rgb(${metadata.color[0]}, ${metadata.color[1]}, ${metadata.color[2]})`
-              : '#636363';
-            const name = users[message.userId];
-            return (
-              <div key={message.date}>
-                {(i === 0 || messages[i - 1].userId !== message.userId) && (
-                  <div
+    <Collapsible defaultOpen title="Chat">
+      <Container direction="vertical" ref={messagesRef}>
+        <Messages>
+          {messages.length > 0 ? (
+            orderedMessages(messages).map((message, i) => {
+              const { color, name, metadata } = messageData(message);
+              return (
+                <Element key={message.date}>
+                  {isNotSameUser(message, i) && (
+                    <Stack
+                      paddingTop={2}
```

Comment:
This one has a padding-top because it's the only I can control if it's a new user that wrote the above message and make the margins 16 instead of 8

:(